starport/pkg/nodetime/sta/sta.go

```diff
@@ -30,6 +30,8 @@ func Generate(ctx context.Context, outPath, specPath, moduleNameIndex string) er
 		dir,
 		"-n",
 		file,
+		// NB: Use axios client since it works for both nodejs and web.
+		"--axios",
```

Comment:
@clockworkgr can you please test the generated Vue store to check if everything works well after this change?

Reply:
the axios change is not a problem (works fine) but see my comments below